deployment/ansible/group_vars/uat/accounts.yml

```diff
@@ -0,0 +1,2 @@
+genesis_accounts:
+  "BKSRJ6RvBkUhW51L15s0v1LyhbLRZndE8RNAWy3BjonpcwZv+7yRYYQ567v7y2aRMwg1Jibue8fwe5SLX3ekArc=": 100000000000001
```

Comment:
@dincho Will the users be able to connect to UAT without modifying the accounts.json in the released package?

Reply:
@lucafavatella see the changes in 112aa5b